src/parameters.toml

```diff
@@ -95,44 +95,51 @@ alias = "Pr_0_Businger"
 value = 0.74
 type = "float"
 description = "Pr_0 for Businger universal functions. From Businger et al, 1971. DOI: 10.1175/1520-0469(1971)028<0181:FPRITA>2.0.CO;2."
+tags = ["Businger", "SurfaceFluxes", "UniversalFunctions"]
```

Comment:
If we put tags in the default file, be aware that this starts to set convention. If so, we should explicitly state the convention in the docs.

I can't quite make out the convention here. It seems to be a struct that will call this value (e.g. `Businger`) but also the components that build this struct `Businger`?), is that the idea?

Reply:
Maybe start conservatively with just the package it pertains to (`SurfaceFluxes`) in this case.

Reply:
I agree, it's best to start with just the package name for these parameters. We can determine conventions when we add tags for the rest of the defaults.